hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleAccount.java

```diff
@@ -93,15 +93,12 @@ public MerkleAccount(final List<MerkleNode> children, final MerkleAccount immuta
 	}
 
 	public MerkleAccount(final List<MerkleNode> children) {
```

Comment:
would this be called during migration form 0.24.0 state ? before we call ServicesState.migrate(), if need a constructor that needs 3 Childs, we will miss the tokens child and thus the migration that we need will not happen right !!

Reply:
When deserializing a v0.24.0 state, the `MerkleDataInputStream` will still read all three children and then pass that list to this constructor. 😁